database/migration/migrateData.ts

```diff
@@ -431,6 +431,19 @@ const migrateData = async () => {
         AND (OLD.is_external_reference_doc = TRUE OR OLD.is_internal_reference_doc = TRUE))
       EXECUTE FUNCTION public.mark_file_for_deletion ();
     `)
+
+    console.log(' - Adding reviewability fields to template_element')
+
+    await DB.changeSchema(`
+    CREATE TYPE public.is_reviewable_status AS ENUM (
+      'ALWAYS',
+      'NEVER' );`)
+
+    await DB.changeSchema(`
+      ALTER TABLE template_element
+        ADD COLUMN IF NOT EXISTS is_reviewable public.is_reviewable_status DEFAULT NULL;
```

Comment:
Thought it best to leave default as NULL as we don't really want to change the functionality of existing templates. For Angola we can just to a bulk database replace if we want their standard to be "ALWAYS".